app/views/recipe_foods/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with(model: [@recipe, @recipe_food], local: true) do |form| %>
+  <div class="field">
+    <%= form.label :food_id, 'Food' %>
+    <%= form.collection_select :food_id, @foods, :id, :name, prompt: 'Select a food' %>
+  </div>
+
+  <div class="field">
+    <%= form.label :quantity %>
+    <%= form.number_field :quantity %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit %>
+  </div>
```

Comment:
- [ ]  Please, in accordance to the project requirement the new recipe form should only be accessible by the logged-in user. I sign-up with another account and was able to create a new food. Kindly correct this.

<img width="587" alt="Untitled" src="https://github.com/lerfast/recipe-app/assets/103386183/1e95c384-f01d-4794-8988-b4577a684c7c">

Reply:
I modify the code for project requires, now the app works as expected.

![image](https://github.com/lerfast/recipe-app/assets/111823606/e0fd6e3a-bc8f-4ce5-8dfa-263c9a8740fb)

Reply:
- Well resolved